test-sbt/jvm/src/main/scala/zio/test/sbt/ZTestRunner.scala

```diff
@@ -133,21 +133,22 @@ case class MergedSpec(spec: ZTestTaskNew, merges: Int)
 class ZTestTaskPolicyDefaultImpl extends ZTestTaskPolicy {
 
   override def merge(zioTasks: Array[ZTestTask]): Array[Task] = {
-    val (newTaskOpt, legacyTaskList) =
+    val (newMergedSpecs, legacyTaskList) =
       zioTasks.foldLeft((List.empty: List[MergedSpec], List[ZTestTaskLegacy]())) {
         case ((newTests, legacyTests), nextSpec) =>
           nextSpec match {
             case legacy: ZTestTaskLegacy => (newTests, legacyTests :+ legacy)
             case taskNew: ZTestTaskNew =>
               newTests match {
                 case existingNewTestTask :: otherTasks =>
-                  if (existingNewTestTask.merges < 1) {
+                  if (true) {
+//                    if (existingNewTestTask.merges < 5) {
```

Comment:
remove?

Reply:
This is marked WIP to avoid premature review. It's not remotely close to being finished; it just needs to trigger Github Actions runs :slightly_smiling_face: